waiter/src/waiter/scheduler.clj

```diff
@@ -665,6 +666,32 @@
                         service
                         (assoc instances :active-instances active-instances))))))))
 
+(defn start-new-services-handler
+  [token-metric-chan-mult start-new-service-fn leader?-fn fallback-state-atom]
```

Comment:
Feels like this component should be able to work of local-usage-agent state and any differences to last execution time of service IDs. Are there any benefits to passing along the data in the mult?

Reply:
It seems simpler/easier to read when using the data in the mult instead of doing a last-request-time diff check on all service-ids on an interval to recalculate the events (that were already calculated)

The diff would need to check if:
- new last request time for a service-id
- service-id maps to a token is actually pointing to a new service-id
- start service for this token

the mult already has the filtered tokens that require a new service to be started